docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,289 @@
+title.VerticalAlignment = VerticalAlignment.Center;
+
+// Add the text to the window
+window.Add(title);
+```
+
+![ NewWindow ](./media/new_window.png)
+
+## Resize window
+
+The size of a window can be retrieved, and the window can be resized:
+
+```csharp
+// Retrieve the current size of the window
+Size2D windowSize = window.WindowSize;
+
+// Resize the window by increasing its width
+windowSize.Width += 100;
+window.WindowSize = windowSize;
+```
+
+When the window size is changed, a resize event is emitted.
+
+## Move window
+
+It is easy to retrieve the current position of a window and move the window to a new position:
+
+```csharp
+// Retrieve the current position of the window
+Position2D windowPosition = window.WindowPosition;
+
+// Moving the window to the right
+windowPosition.X += 100;
+window.WindowPosition = windowPosition;
+```
+
+## Show and hide window
+
+When a window is created, by default it is in visible state. It is also possible to hide a window by making it invisible:
+
+```csharp
+// Hide the window
+window.Hide();
+
+// Show the window
+window.Show();
+```
+
+If all the windows are hidden, updating and rendering of the windows is paused.
+
+## Event handling
+
+It allows you to easily handle various events emitted by the window.
+The following are some default events:
+
+### Touch event
+
+If you touch the window, the touch event is triggered. If multi points are touched, touch event is triggered when the first touch occurs and then when the last finger is lifted.
+
+The following code shows how to handle the touch event:
+
+```csharp
+window.TouchEvent += OnWindowTouched;
+
+private void OnWindowTouched(object sender, Window.TouchEventArgs e)
+{
+    if (e.Touch.GetState(0) == PointStateType.Down)
+    {
+        // The window has been touched, do something
+    }
+}
+```
+
+### Key event
+
+Key event is emitted when the window receives a key event from the window manager.
+
+The following code shows how to handle the key event:
+
+```csharp
+window.KeyEvent += OnWindowKeyEvent;
+
+private void OnWindowKeyEvent(object sender, Window.KeyEventArgs e)
+{
+    if (e.Key.State == Key.StateType.Down)
+    {
+        if (e.Key.KeyPressedName == "Left")
+        {
+            // The left arrow key is pressed, do something
+        }
+        else if (e.Key.KeyPressedName == "Right")
+        {
+            // The right arrow key is pressed, do something
+        }
+    }
+}
+```
+
+### Resize event
+
+Resize event is emitted when the window is resized.
+
+The following code shows how to handle the resize event:
+
+```csharp
+window.Resized += OnWindowResized;
+
+private void OnWindowResized(object sender, Window.ResizedEventArgs e)
+{
+    // Window is resized, do something
+}
+```
+
+For more information, see [Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) API.
+
+## Window rotation
+
+To support rotation, NUI provides two methods.
+// To do: tscholb
+
+### Enable or disable window
```

Comment:
@surbhig12 I want to ask your opinion about this heading.
I reverted back to "window", but not sure whether it is proper word for this section.
According to the content, still it seems it is not containing the exact meaning.

In my opinion, another suggestion can be below. Would you please let me know your opinion also?

\### Enable or disable orientations